src/valid/mod.rs

```diff
@@ -94,6 +94,7 @@ pub struct Validator {
     types: Vec<r#type::TypeInfo>,
     layouter: Layouter,
     location_mask: BitSet,
+    built_ins: FastHashSet<crate::BuiltIn>,
```

Comment:
Suggestion: we could probably just use a `BitSet` here, just as we do for `location_mask`, and cast `BuiltIn` to `usize` when we work with it. (The C programmer in me wants a `u32`, but that's not the way we do things any more...)

Reply:
oh indeed, I'll do this